src/v2/pages/who-you-own/who-you-own.tsx

```diff
@@ -77,6 +78,33 @@ export const WhoYouOwn = () => {
         setOpenCharacterItemDialog(false);
     };
 
+    // For the power graph
+    const teamPowerData: { x: string; y: number }[] = [];
+    const teamAttributeData: { x: string; y: number }[] = [];
+    const teamAbilityData: { x: string; y: number }[] = [];
+    charactersFiltered.forEach(character => {
+        const power = CharactersPowerService.getCharacterPower(character);
+        const attributePower = CharactersPowerService.getCharacterAttributePower(character);
+        const abilityPower = CharactersPowerService.getCharacterAbilityPower(character);
+
+        teamPowerData.push({ x: character.name, y: power });
+        teamAttributeData.push({ x: character.name, y: attributePower });
+        teamAbilityData.push({ x: character.name, y: abilityPower });
+    });
+
+    const sortByPower = (a: { x: string; y: number }, b: { x: string; y: number }) => b.y - a.y;
+    teamPowerData.sort(sortByPower);
+    teamAttributeData.sort((a, b) => {
+        const aIndex = teamPowerData.findIndex(item => item.x === a.x);
+        const bIndex = teamPowerData.findIndex(item => item.x === b.x);
+        return sortByPower(teamPowerData[aIndex], teamPowerData[bIndex]);
+    });
+    teamAbilityData.sort((a, b) => {
+        const aIndex = teamPowerData.findIndex(item => item.x === a.x);
+        const bIndex = teamPowerData.findIndex(item => item.x === b.x);
+        return sortByPower(teamPowerData[aIndex], teamPowerData[bIndex]);
+    });
+
```

Comment:
This could be part of team graph component as well, pass `charactersFiltered` as component prop

Reply:
I'm thinking it will be better to leave on this page and allow for multiple sorting orders, depending on the view order selected. Sort by power (as it is currently) when isCharactersView is true, or order by faction when isFactionsView is true.